webrecorder/browsertrix
# Allow users to set crawl config language

Support setting [ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) in crawl. Resolves https://github.com/webrecorder/browsertrix-cloud/issues/368.

### Manual testing
1: Install new dependency `yarn`
2. Run app `yarn start`
3. Go to new crawl template creation view. Verify "Language" dropdown is shown in "Crawl Settings" section with correct options
4. Choose language and save. Verify language shows on crawl template detail view.
5. Click "Edit" in "Configuration" section and update language. Verify language saves as expected.

### Screenshots
**Language select in New Crawl Template view:**
<img width="971" alt="Screen Shot 2022-11-21 at 11 27 54 AM" src="https://user-images.githubusercontent.com/4672952/203142903-56a09fbd-bf2f-4abe-ab12-bf1e19f9c563.png">

**Dropdown options:**
<img width="625" alt="Screen Shot 2022-11-21 at 11 27 59 AM" src="https://user-images.githubusercontent.com/4672952/203142951-933f0888-5a0d-4ba8-a182-a18f7c2513bd.png">

**Tooltip help text:**
<img width="340" alt="Screen Shot 2022-11-21 at 11 28 06 AM" src="https://user-images.githubusercontent.com/4672952/203142995-837c5aba-7a38-4772-897f-9145a3922a73.png">

**Language field in Crawl Template detail view:**
<img width="901" alt="Screen Shot 2022-11-21 at 11 28 27 AM" src="https://user-images.githubusercontent.com/4672952/203143077-13ea0b6d-da7a-4f0e-a4d9-c2c26fce2b2f.png">

**Language select in Crawl Template edit view:**
<img width="572" alt="Screen Shot 2022-11-21 at 11 28 37 AM" src="https://user-images.githubusercontent.com/4672952/203143124-1b4a50c3-039a-4118-98e4-98f8c2463925.png">
